python-lib/ontology_tagging/ontology_tagging_formatting.py

```diff
@@ -126,477 +102,9 @@ def _set_columns_order(
         """Concatenate the input_df with the new one,reset its columns in the right order, and return it"""
         input_df = input_df.drop(columns=self.text_column_tokenized)
         df = pd.concat([input_df, output_df], axis=1)
+        df = df.drop_duplicates()
```

Comment:
This drops duplicate rows, not columns. Is this expected ?

Reply:
Yes, I've added this line after Krishna have tested the plugin, and it is supposed to remove rows in the case we have the same keyword that appears multiple times in the same sentence, which create identical rows